src/labels/CustomLabels.labels

```diff
@@ -284,35 +580,67 @@
         <fullName>stgDefaultFacultyTypeTitle</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
-        <protected>false</protected>
```

Comment:
I agree with @ceiroa, that it is a good idea to make all new labels protected. Unfortunately, we cannot change existing labels due to packaging restrictions. So we'll need to change these back.

Reply:
@davidjray 

I see I will change those back, thanks!